domains/schulsani.is-local.org.json

```diff
@@ -0,0 +1,17 @@
+{
+    "description": "Schulsani Projekt",
+
+    "domain": "is-local.org",
+    "subdomain": "schulsani",
+
+    "owner": {
+        "repo": "https://github.com/thetechnikfreak/register1",
+        "email": "contact@thetechnikfreak.is-a.dev"
+    },
+
+    "record": {
+        "NS": ["ns1.desec.io", "ns2.desec.org"],
```

Comment:
We no longer support NS records as per the readme.md:

![image](https://github.com/open-domains/register/assets/76603072/328afc3e-52a6-4f5a-b6a6-d2e42328bf99)